Niamh25-DOC-4895-ChangethecommandinConfiguringaJPAnotarybackend (#1150)
corda/corda-docs-portal: content/en/platform/corda/4.8/enterprise/notary/installing-jpa.md

### Connection string

The properties specifying the location of the client certificates must be passed in via the JDBC connection

> string. It will not be possible to pass them in as configuration properties. See below for an example connection string.

Reviewer: ```suggestion
string. It is not possible to pass them in as configuration properties. See below for an example connection string.
```

> string. It will not be possible to pass them in as configuration properties. See below for an example connection string.

Reviewer 2: ```suggestion
string. It is not be possible to pass them in as configuration properties. See below for an example connection string.
```

{{< note >}}

> You must have a minimum of 3 nodes. Each node in the connection string should have the 26257 port number at the end of it.

Reviewer: ```suggestion
The dataSource.url property must specify a _minimum_ of 3 nodes. Each node in the connection string must end with the port number 26257.
```

Reviewer 2: Sorry, Paul and I have overlapped. I prefer his first sentence here :)

> You must have a minimum of 3 nodes. Each node in the connection string should have the 26257 port number at the end of it.

Reviewer 2: ```suggestion
You must specify a minimum of three nodes. The connection string of each node must specify the port number as 26257.
```

> The dataSource.url property must specify a _minimum_ of 3 nodes. Each node in the connection string must end with the port number 26257.

Reviewer 2: ```suggestion
The `dataSource.url` property must specify a _minimum_ of three nodes. Each node in the connection string must end with port number 26257.
```

{{</ note >}}

```javascript

> dataSource.url="jdbc:postgresql://{CockroachDB node1 IP address}:26257,{CockroachDB node2 IP address}:26257,{CockroachDB node3 IP address}:26257/corda?sslmode=require&sslrootcert=certificates/ca.crt&sslcert=certificates/client.corda.crt&sslkey=certificates/client.corda.key.pk8"

Reviewer 2: ```suggestion
dataSource.url="jdbc:postgresql://<CockroachDB-node1-IP-address>:26257,<CockroachDB-node2-IP-address>:26257,<CockroachDB-node3-IP-address>:26257/corda?sslmode=require&sslrootcert=certificates/ca.crt&sslcert=certificates/client.corda.crt&sslkey=certificates/client.corda.key.pk8"
```
